addons/website/static/src/js/menu/content.js

```diff
@@ -398,6 +407,19 @@ var PagePropertiesDialog = weWidgets.Dialog.extend({
     _onVisibilityChanged: function (ev) {
         this.$('.show_visibility_password').toggleClass('d-none', ev.target.value !== 'password');
         this.$('.show_visibility_group').toggleClass('d-none', ev.target.value !== 'restricted_group');
+        this.$('#visibility_password').attr('required', ev.target.value === 'password');
+    },
+    /**
+     * @private
+     */
+    _onPasswordClicked: function (ev) {
+        this.$('#visibility_password').val('');
+        this._onPasswordChanged();
+    },    /**
+     * @private
+     */
+    _onPasswordChanged: function (ev) {
+        this.$('#visibility_password').data('dirty', 1);
```

Comment:
this flag is not used, I guess the intention was to use it to decide if we return `visibility_password` or not